doc/dev/decisionrecords/Codestyle.md

```diff
@@ -99,43 +99,44 @@ Working Directory: $ProjectFileDir$
 
 ### Other IDEs
 
-We do not have support for other IDEs at the moment. If you use another editor and make one please
+We do not have support for other IDEs at the moment. If you use another editor and make one, please
 feel free to share it.
 
 ### Sorting Class Members
 
-Some of the classes in OTP have a lot of fields and methods. Keeping members sorted reduce the merge
+Some of the classes in OTP have a lot of fields and methods. Keeping members sorted reduces merge
 conflicts. Adding fields and methods to the end of the list will cause merge conflicts more often
-than inserting methods and fields in an ordered list. Fields and methods can be sorted in "feature"
-sections or alphabetically, but stick to it and respect it when adding new methods and fields.
+than will inserting methods and fields in an ordered list. Fields and methods can be sorted in
+"feature" sections or alphabetically, but stick to it and respect it when adding new methods and
+fields.
 
 The provided formatter will group class members in this order:
 
-1. Getter and Setter methods are kept together
-2. Overridden methods are kept together
-3. Dependent methods are sorted in a breadth-first order.
+1. Getter and setter methods are kept together.
+2. Overridden methods are kept together.
+3. Dependent methods are sorted in breadth-first order.
 4. Members are sorted like this:
-    1. `static` `final` fields (constants)
+    1. `static final` fields (constants)
     2. `static` fields (avoid)
-    3. instance fields
-    4. static initializer
-    5. class initializer
-    6. constructor
+    3. Instance fields
+    4. Static initializers
+    5. Class initializers
+    6. Constructors
     7. `static` factory methods
     8. `public` methods
-    9. getter and setters
-    8. `private`/package methods
-    10. `private` enums (avoid `public`)
-    11. interfaces
-    12. `private static` classes (avoid `public`)
-    13. instance classes (avoid)
+    9. Getter and setters
+    10. `private`/package methods
+    11. `private` enums (avoid `public`)
+    12. Interfaces
+    13. `private static` classes (avoid `public`)
+    14. Instance classes (avoid)
```

Comment:
Looks like when I applied https://github.com/opentripplanner/OpenTripPlanner/pull/6125#discussion_r1795547119, it somehow messed up the order numbers.